cmd/stack_create_test.go

```diff
@@ -150,21 +165,26 @@ func TestStackCreateInvalidStackCase4(t *testing.T) {
 	sandbox, cleanup := cmdtest.TestSetupWithSandbox(t, true)
 	defer cleanup()
 
-	err := os.RemoveAll("testing-stack")
-	if err != nil {
-		t.Fatal(err)
-	}
+	testStackName := "testing-stack-create-invalid-4"
 
-	args := []string{"stack", "create", "testing-stack", "--copy", "exp/java-microprofile"}
-	_, err = cmdtest.RunAppsody(sandbox, args...)
+	args := []string{"stack", "create", testStackName, "--copy", "exp/java-microprofile"}
+	_, err := cmdtest.RunAppsody(sandbox, args...)
 
 	if err == nil {
 		t.Fatal(err)
```

Comment:
A better message here would be nice as to why the test is failing. The `err` will be `nil` at this point.